hooks/go/030-remove-sc-and-secrets-on-module-delete/remove-sc-and-secrets-on-module-delete.go

```diff
@@ -0,0 +1,117 @@
+package hooks_common
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	snapv1 "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumesnapshot/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	sv1 "k8s.io/api/storage/v1"
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/deckhouse/csi-ceph/api/v1alpha1"
+	"github.com/deckhouse/csi-ceph/hooks/go/consts"
+	"github.com/deckhouse/module-sdk/pkg"
+	"github.com/deckhouse/module-sdk/pkg/registry"
+	"github.com/deckhouse/sds-common-lib/kubeclient"
+)
+
+
+func removeFinalizers(ctx context.Context, cl client.Client, obj client.Object, logger pkg.Logger) error {
```

Comment:
Removing all the finalizers generally looks dangerous. Shouldn't we pass a list of finalizers we know are safe to delete?

Reply:
No. There all finalizers wiped for final objects deletion. So there must be no finalizers at all after hook finish work.